src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  list-style: none;
+  padding-top: 5px;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 4px 0;
+}
+
+.stars--4 span:nth-child(-n + 4)::before {
+  content: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars__reviews {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  padding-left: 15px;
+  display: flex;
+  align-items: center;
+}
```

Comment:
reviews is a part of card, stars block should only contain stars

Reply:
reviews amended